packages/script-writer/src/ScriptWriter.ts

```diff
@@ -0,0 +1,45 @@
+import { agentFunctions } from "./agent-functions";
+import { ResultErr } from "@polywrap/result";
+import { IAgent, Workspace } from "@evo-ninja/agent-utils";
+import { LlmApi, Chat, Logger, StepOutput, RunResult, executeAgentFunction, basicFunctionCallLoop } from "@evo-ninja/agent-utils";
+import { AgentContext } from "./AgentContext";
+import { GOAL_PROMPT, INITIAL_PROMP, LOOP_PREVENTION_PROMPT } from "./prompts";
+
+export class ScriptWriter implements IAgent {
```

Comment:
Can we do away with the `I` prefix for interfaces? I think it is redundant, isn't consistent with the rest of the codebase, and is something that is not very common in modern typescript. GPT response: https://chat.openai.com/share/fe121cf1-d304-4aad-a223-151ae99ec458

Reply:
While I would much rather work on a project with this convention than without, and argued for such in the past, it is not a hill I want to die on, so sure, we can do away with the convention on evo.

Reply:
Bless your soul :pray: I would honestly be fine with it if I saw other projects do the same within the TypeScript ecosystem.